corehq/apps/style/static/style/less/bootstrap3/forms.less

```diff
@@ -94,6 +94,11 @@ legend .subtext {
   }
 }
 
+// Static width for select2 widgets, which otherwise grow too large on case management pages
+.form-control.select2-container {
```

Comment:
limited to `.app-manager-content`, but left it in forms.less because I suspect this is going to come up in other B3 select2 widgets